### DISCUSSION TITLE
Cannot downscale pixel art image.

### DISCUSSION THREAD

**Discussion by pegasusearl** [ImageMagick/ImageMagick#6788] [Help] [answered]
### ImageMagick version

7.1.1-15 Q16-HDRI x86_64 21298

### Operating system, version and so on

Manjaro Linux x86_64, 6.1.53-1-MANJARO

### Description

I'm trying to downscale this image to 50% of it's size, I  wanted to not use filtering because it's pixel art. And I wanted to use ImageMagick since there are many files and many subfolders.

![handtouch1](https://github.com/ImageMagick/ImageMagick/assets/23167087/d4457845-1535-4818-8251-5c7297d5beb2)

I expect the result to be like this (I downscaled it with krita, with "nearest" filter)
![handtouch1krit](https://github.com/ImageMagick/ImageMagick/assets/23167087/5dc94754-7435-4827-8db6-ae8a4c68b71d)

I have read the documentations and forum posts then came up with these commands. Here are the commands I tried, all of them didn't work like what I expected.

`mogrify -resize 50% -filter Box handtouch1ds.png`
![handtouch1ds](https://github.com/ImageMagick/ImageMagick/assets/23167087/830ea6e8-14ea-436e-ba9c-63eb4e71d625)

`mogrify -resize 50% +antialias handtouch1ds.png`
![handtouch1ds](https://github.com/ImageMagick/ImageMagick/assets/23167087/b0d89d80-b7d7-432b-954c-b569eb17ca42)

`convert handtouch1.png -antialias -scale 50% handtouch1dt.png`
![handtouch1dt](https://github.com/ImageMagick/ImageMagick/assets/23167087/f7e985f2-5615-44e9-a9f1-74672c2bad6a)

`convert handtouch1.png +antialias -scale 50% handtouch1dt.png`
![handtouch1dt](https://github.com/ImageMagick/ImageMagick/assets/23167087/e8e4cb6f-8bcd-43eb-a300-146cf02be853)

Maybe I misunderstand on how it worked. What would be the correct command to do what I wanted?
which is downscale 50% with "nearest neighbor" filter, or no filter at all?

**Comment by fmw42** [comment, marked as answer]
Try -sample

**Comment by snibgo** [comment]
The input is 180 pixels high. Downscaling to 50%, we expect the height will be 90 pixels. That is what IM does, with various methods. But the "expected result" from Krita is 91 pixels high.

EDIT: This command gives exactly the "required result":
```
magick finger.png -filter point -resize 50.5% out.png
```

**Comment by pegasusearl** [comment]
Thank you, yes both worked. I didn't really care much about the exact size, just the filtering.
